tests/com/carolinarollergirls/scoreboard/DefaultSettingsModelTests.java

```diff
@@ -1,7 +1,6 @@
 package com.carolinarollergirls.scoreboard;
 
-import static org.junit.Assert.assertNull;
-import static org.junit.Assert.assertSame;
+import static org.junit.Assert.*;
```

Comment:
You and @frank-weinberg are changing these in different directions. Can ye decide which ye're going with?

Reply:
Did ye two decide which way to go on these?

Reply:
I'm fine with @frank-weinberg's approach with specifically-declared imports instead of `*` imports, I've updated this PR to match.